### PR TITLE
Bromley waste phase2 bugfixes 2

### PR DESCRIPTION
**Pull request by struan**
[skip changelog]

### CODE REVIEW

File: perllib/FixMyStreet/App/Controller/Waste.pm [hunk 1]
```diff
@@ -1082,7 +1083,7 @@ sub process_garden_renew : Private {
         if ( $c->stash->{is_staff} ) {
             $c->forward('csc_code');
         } elsif ( $payment_method eq 'direct_debit' ) {
-            $c->forward('direct_debit_renew');
+            $c->forward('direct_debit');
```
Comment (dracos) [inline perllib/FixMyStreet/App/Controller/Waste.pm:1086]: Can the direct_debit_renew function be removed then?

**Review by dracos — CHANGES_REQUESTED** [review]
Few tiny things

File: perllib/FixMyStreet/App/Form/Waste/Garden/Modify.pm [hunk 2]
```diff
@@ -84,7 +84,6 @@ has_field task => (
     widget => 'RadioGroup',
     options => [
         { value => 'modify', label => 'Increase or reduce the number of bins in your subscription' },
-        { value => 'problem', label => 'Request a replacement for a broken or stolen bin' },
```
[the remark this answers is not in the document]
Comment (struan) [reply perllib/FixMyStreet/App/Form/Waste/Garden/Modify.pm:87]: I was sure I had :(

**Review by dracos — APPROVED** [review]

**Comment by dracos** [comment]
@struan Well approved apart from the tests are failing :)

**Pull request merged**